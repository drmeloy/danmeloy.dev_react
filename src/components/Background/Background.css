.Background {
  width: 100%;
  min-height: 101vh;
  padding-left: 130px;
  padding-top: 1%;

  p {
    font-family: 'Raleway', sans-serif;
  }

  .Title {
    font-size: 1.5em;
    letter-spacing: 4px;
    font-weight: bold;
    margin-left: 2vw;
    margin-bottom: .6em;
  }

  .Blurb {
    padding: 0 0 1% 3% ;
    font-weight: bold;
    font-style: italic;
  }

  .Titles {
    display: flex;
    justify-content: space-between;
    text-align: center;
    font-size: 1em;
    font-family: 'Raleway', sans-serif;
    margin: 0;
    padding: 1vw 3vw;

    h4 {
      width: 20vw;
    }
  }

  .Excerpts {
    display: flex;

    p {
      padding: 0 3vw;
      font-size: 1em;
      line-height: 1.8rem;
    }
  }

  .Closer {
    font-size: 1.2em;
    width: 70vw;
    text-align: right;
    font-weight: bold;
    padding: 2vw 3vw;
    position: relative;
    left: 20%;
    font-style: italic;
  }

  @media (min-width: 10px) and (max-width: 1025px){
    display: none;
  }

  background: linear-gradient(52deg, rgba(163, 163, 163, 0.09) 0%, rgba(163, 163, 163, 0.09) 33.3%,rgba(100, 100, 100, 0.09) 33.3%, rgba(100, 100, 100, 0.09) 66.6%,rgba(162, 162, 162, 0.09) 66.6%, rgba(162, 162, 162, 0.09) 99%),linear-gradient(258deg, rgba(193, 193, 193, 0.06) 0%, rgba(193, 193, 193, 0.06) 33.3%,rgba(169, 169, 169, 0.06) 33.3%, rgba(169, 169, 169, 0.06) 66.6%,rgba(92, 92, 92, 0.06) 66.6%, rgba(92, 92, 92, 0.06) 99%),linear-gradient(129deg, rgba(45, 45, 45, 0.03) 0%, rgba(45, 45, 45, 0.03) 33.3%,rgba(223, 223, 223, 0.03) 33.3%, rgba(223, 223, 223, 0.03) 66.6%,rgba(173, 173, 173, 0.03) 66.6%, rgba(173, 173, 173, 0.03) 99%),linear-gradient(280deg, rgba(226, 226, 226, 0.06) 0%, rgba(226, 226, 226, 0.06) 33.3%,rgba(81, 81, 81, 0.06) 33.3%, rgba(81, 81, 81, 0.06) 66.6%,rgba(186, 186, 186, 0.06) 66.6%, rgba(186, 186, 186, 0.06) 99%),linear-gradient(85deg, rgba(225, 225, 225, 0.04) 0%, rgba(225, 225, 225, 0.04) 33.3%,rgba(95, 95, 95, 0.04) 33.3%, rgba(95, 95, 95, 0.04) 66.6%,rgba(39, 39, 39, 0.04) 66.6%, rgba(39, 39, 39, 0.04) 99%),linear-gradient(128deg, rgba(184, 184, 184, 0.06) 0%, rgba(184, 184, 184, 0.06) 33.3%,rgba(0, 0, 0, 0.06) 33.3%, rgba(0, 0, 0, 0.06) 66.6%,rgba(140, 140, 140, 0.06) 66.6%, rgba(140, 140, 140, 0.06) 99.89999999999999%),linear-gradient(323deg, rgba(40, 40, 40, 0.07) 0%, rgba(40, 40, 40, 0.07) 33.3%,rgba(214, 214, 214, 0.07) 33.3%, rgba(214, 214, 214, 0.07) 66.6%,rgba(190, 190, 190, 0.07) 66.6%, rgba(190, 190, 190, 0.07) 99.89999999999999%),linear-gradient(61deg, rgba(230, 230, 230, 0) 0%, rgba(230, 230, 230, 0) 33.3%,rgba(241, 241, 241, 0) 33.3%, rgba(241, 241, 241, 0) 66.6%,rgba(55, 55, 55, 0) 66.6%, rgba(55, 55, 55, 0) 99%),linear-gradient(0deg, #2625e3,#0bbaef)
}