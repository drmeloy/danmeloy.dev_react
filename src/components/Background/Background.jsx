import React from 'react';
import styles from './Background.css';

export default function Background(){
  return (
    <>
      <section className={styles.Background}>
        <p className={styles.Title}>A bit about my background...</p>
        <div className={styles.Titles}>
          <h4 className={styles.One}>Perspectives-based approach</h4>
          <h4>International entrepreneur</h4>
          <h4>Leadership and empowering others</h4>
        </div>
        <div className={styles.Excerpts}>
          <p>My life has provided me with a range of experiences that help shape who I am. As a student of Anthropology, I learned how perspectives often vary and hugely differentiate the way we understand the world. Throughout my life I strive to keep a perspectives-based approach to problem solving and relationship building, believing that the first step to success is understanding others as best we can.</p>
          <p>I&apos;ve opened two successful businesses abroad in Quito, Ecuador. Being a co-owner and founder of <a href='https://communityhostel.com/' target='_blank' rel='noopener noreferrer'>Community Hostel</a> and <a href='http://bandidobrewing.com/en/' target='_blank' rel='noopener noreferrer'>Bandido Brewing</a> taught me how to take ideas and turn them into reality. The challenges of opening two businesses in a foreign country taught me the value of adaptability. With a mindset of adaptability, any problem is solvable, any issue or interruption is merely a part of the process. To this day my adaptability remains one of my greatest strengths.</p>
          <p>I&apos;ve been a leader and facilitator for a wilderness non-profit. I led backcountry wilderness trips for adults with physical and developmental disabilities and facilitated outdoor experiential education events for under-served student communities. By helping people connect with the outdoors I was helping them build their confidence and realize that they can do anything they set their minds to, regardless of personal background or ability level.</p>
        </div>
        <p className={styles.Closer}>Now, as a developer and engineer, I hope to use technology to apply the lessons I&apos;ve learned throughout my life. I aim to solve problems that can empower others.</p>
      </section>
    </>
  );
}
