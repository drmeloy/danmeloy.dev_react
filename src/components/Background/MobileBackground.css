.Background {
  display: none;
  font-weight: bold;
  font-family: 'Raleway', sans-serif;

  .Title {
    font-size: 1rem;
    text-align: center;
  }

  .Blurb {
    text-align: center;
    font-size: .7rem;
    width: 80vw;
    margin: 3% auto;
    font-style: italic;
  }

  .Excerpts {
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    height: 50%;
    padding: 1em;

    .Detail {
      margin-bottom: 1em;
      font-size: .8rem;
    }

    .Excerpt {
      padding: 1em 1em;
      padding-bottom: 0;
      font-size: .8rem;
      text-align: center;
    }
  }

  .Closer {
    font-size: .8rem;
    width: 80vw;
    text-align: center;
    margin: auto;
    font-style: italic;
  }

  @media (min-width: 720px) and (max-width: 1025px) {
    .Title {
      font-size: 2rem;
    }

    .Blurb {
      font-size: 1rem;
    }

    .Excerpts {
      .Detail, .Excerpt {
        font-size: 1.2rem;
      }

      .Excerpt {
        width: 80%;
        margin: auto;
      }
    }

    .Closer {
      font-size: 1.2rem;
    }
  }

  @media (min-width: 10px) and (max-width: 1025px) {
    display: block;
    min-height: 101vh;
    padding-top: 3vh;
    width: 100%;
  }

  background: linear-gradient(52deg, rgba(163, 163, 163, 0.09) 0%, rgba(163, 163, 163, 0.09) 33.3%,rgba(100, 100, 100, 0.09) 33.3%, rgba(100, 100, 100, 0.09) 66.6%,rgba(162, 162, 162, 0.09) 66.6%, rgba(162, 162, 162, 0.09) 99%),linear-gradient(258deg, rgba(193, 193, 193, 0.06) 0%, rgba(193, 193, 193, 0.06) 33.3%,rgba(169, 169, 169, 0.06) 33.3%, rgba(169, 169, 169, 0.06) 66.6%,rgba(92, 92, 92, 0.06) 66.6%, rgba(92, 92, 92, 0.06) 99%),linear-gradient(129deg, rgba(45, 45, 45, 0.03) 0%, rgba(45, 45, 45, 0.03) 33.3%,rgba(223, 223, 223, 0.03) 33.3%, rgba(223, 223, 223, 0.03) 66.6%,rgba(173, 173, 173, 0.03) 66.6%, rgba(173, 173, 173, 0.03) 99%),linear-gradient(280deg, rgba(226, 226, 226, 0.06) 0%, rgba(226, 226, 226, 0.06) 33.3%,rgba(81, 81, 81, 0.06) 33.3%, rgba(81, 81, 81, 0.06) 66.6%,rgba(186, 186, 186, 0.06) 66.6%, rgba(186, 186, 186, 0.06) 99%),linear-gradient(85deg, rgba(225, 225, 225, 0.04) 0%, rgba(225, 225, 225, 0.04) 33.3%,rgba(95, 95, 95, 0.04) 33.3%, rgba(95, 95, 95, 0.04) 66.6%,rgba(39, 39, 39, 0.04) 66.6%, rgba(39, 39, 39, 0.04) 99%),linear-gradient(128deg, rgba(184, 184, 184, 0.06) 0%, rgba(184, 184, 184, 0.06) 33.3%,rgba(0, 0, 0, 0.06) 33.3%, rgba(0, 0, 0, 0.06) 66.6%,rgba(140, 140, 140, 0.06) 66.6%, rgba(140, 140, 140, 0.06) 99.89999999999999%),linear-gradient(323deg, rgba(40, 40, 40, 0.07) 0%, rgba(40, 40, 40, 0.07) 33.3%,rgba(214, 214, 214, 0.07) 33.3%, rgba(214, 214, 214, 0.07) 66.6%,rgba(190, 190, 190, 0.07) 66.6%, rgba(190, 190, 190, 0.07) 99.89999999999999%),linear-gradient(61deg, rgba(230, 230, 230, 0) 0%, rgba(230, 230, 230, 0) 33.3%,rgba(241, 241, 241, 0) 33.3%, rgba(241, 241, 241, 0) 66.6%,rgba(55, 55, 55, 0) 66.6%, rgba(55, 55, 55, 0) 99%),linear-gradient(0deg, #2625e3,#0bbaef)
}