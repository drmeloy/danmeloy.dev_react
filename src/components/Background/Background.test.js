import React from 'react';
import { shallow } from 'enzyme';
import Background from './Background';

describe('Background component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Background />);
    expect(wrapper).toMatchSnapshot();
  });
});
