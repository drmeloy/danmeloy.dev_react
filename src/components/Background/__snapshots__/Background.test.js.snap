// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background component matches snapshot 1`] = `
<Element
  id="Background"
  name="Background"
>
  <section
    className="Background"
  >
    <p
      className="Title"
    >
      A bit about my background...
    </p>
    <p
      className="Blurb"
    >
      Some context on my approach to life and work:
    </p>
    <div
      className="Titles"
    >
      <h4
        className="One"
      >
        Perspectives-based approach
      </h4>
      <h4>
        International entrepreneur
      </h4>
      <h4>
        Leadership and empowering others
      </h4>
    </div>
    <div
      className="Excerpts"
    >
      <p>
        My life has provided me with a range of experiences that help shape who I am. As a student of Anthropology, I learned how perspectives often vary and hugely differentiate the way we understand the world. Throughout my life I strive to keep a perspectives-based approach to problem solving and relationship building, believing that the first step to success is understanding others as best we can.
      </p>
      <p>
        I've opened two successful businesses in Quito, Ecuador. Being a co-owner and founder of 
        <a
          href="https://communityhostel.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Community Hostel
        </a>
         and 
        <a
          href="http://bandidobrewing.com/en/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Bandido Brewing
        </a>
         taught me how to take ideas and make them reality. The challenges of opening two businesses in a foreign country taught me the value of adaptability. With a mindset of adaptability, any problem is solvable, any issue or interruption is merely a part of the process. To this day my adaptability remains one of my greatest strengths.
      </p>
      <p>
        I've been a leader and facilitator for a wilderness non-profit. I led backcountry wilderness trips for adults with physical and developmental disabilities and facilitated outdoor experiential education events for under-served student communities. By helping people connect with the outdoors I was helping them build their confidence and realize that they can do anything they set their minds to, regardless of personal background or ability level.
      </p>
    </div>
    <p
      className="Closer"
    >
      Now, as a developer and engineer, I use technology to apply the lessons I've learned throughout my life. I work to solve problems that will empower others.
    </p>
  </section>
  <section
    className="Background"
  >
    <p
      className="Title"
    >
      A bit about my background...
    </p>
    <p
      className="Blurb"
    >
      Expand the sections below to learn some context about my approach to life and work:
    </p>
    <div
      className="Excerpts"
    >
      <details
        className="Detail"
      >
        <summary>
          Perspectives-based approach
        </summary>
        <p
          className="Excerpt"
        >
          As a student of Anthropology, I learned how perspectives often vary and hugely differentiate the way we understand the world. Throughout my life I strive to keep a perspectives-based approach to problem solving and relationship building, believing that the first step to success is understanding others as best we can.
        </p>
      </details>
      <details
        className="Detail"
      >
        <summary>
          International entrepreneur
        </summary>
        <p
          className="Excerpt"
        >
          I've opened two successful businesses in Quito, Ecuador. Being a co-owner and founder of 
          <a
            href="https://communityhostel.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Community Hostel
          </a>
           and 
          <a
            href="http://bandidobrewing.com/en/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Bandido Brewing
          </a>
           taught me how to take ideas and make them reality. The challenges of opening two businesses in a foreign country taught me the value of adaptability. With a mindset of adaptability, any problem is solvable, any issue or interruption is merely a part of the process. To this day my adaptability remains one of my greatest strengths.
        </p>
      </details>
      <details
        className="Detail"
      >
        <summary>
          Leadership and empowering others
        </summary>
        <p
          className="Excerpt"
        >
          I've been a leader and facilitator for a wilderness non-profit. I led backcountry wilderness trips for adults with physical and developmental disabilities and facilitated outdoor experiential education events for under-served student communities. By helping people connect with the outdoors I was helping them build their confidence and realize that they can do anything they set their minds to, regardless of personal background or ability level.
        </p>
      </details>
    </div>
    <p
      className="Closer"
    >
      Now, as a developer and engineer, I use technology to apply the lessons I've learned throughout my life. I work to solve problems that will empower others.
    </p>
  </section>
</Element>
`;
